latch_cli/main.py

```diff
@@ -549,18 +560,23 @@ def execute(
     help="Path to a nextflow script to register.",
 )
 @click.option(
-    "--redownload-dependencies",
-    type=bool,
-    is_flag=True,
-    default=False,
-    help="Redownload external Nextflow dependencies",
+    "--nf-version",
+    type=str,
+    default=None,
+    help="Version of Nextflow runtime to use",
```

Comment:
id hardcode this and change it in new latch versions

i.e. if we want to roll out changes to the nf binary we would need to make a new version of latch